src/ComplaintDetail.jsx

```diff
@@ -75,7 +75,8 @@ export class ComplaintDetail extends React.Component {
     return (
       <button className="a-btn a-btn__link"
               onClick={this.props.onClickedBack}>
-          <span className="cf-icon cf-icon-left"></span>
+          {/* Includes custom cf-icon-left class to set color. */}
```

Comment:
It's probably worth factoring out this svg into it's own function since it's duplicated below